distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -89,6 +112,8 @@
 
     private static final String DEFAULT_SAMPLE_PRODUCT_FILE_NAME = "sample.txt";
 
+    private static final DynamicPort RESTITO_STUB_SERVER_PORT = new DynamicPort(41300);
```

Comment:
Will this require this port or will it try to get it and then move on to the next available if it can't get it? I know we've made a lot of strides on making ports dynamic throughout our itests.